recipe/meta.yaml

```diff
@@ -1,67 +1,62 @@
 {% set name = "neuralprophet" %}
-{% set version = "0.3.2" %}
+{% set version = "0.4.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 build:
-  number: 1
-  # lunarcalendar dependencies currently aren't availabl on s390x
-  skip: true  # [py<36 or win32 or s390x]
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  # lunarcalendar dependencies currently aren't available on s390x
```

Comment:
lunarcalendar is available on s390x

Reply:
Tried removing the skip and got the following prefect error https://cloud.prefect.io/anaconda/flow-run/74f88eb6-e5a4-45ce-9584-fc3ec603dd31?logs=&logId=11a2b1ff-7859-4668-a9f2-d7a4af696679

Reply:
Do you need to build it for s390x? If so, the next step is to find the problematic dependency

Reply:
The client does not need a build `s390x`. We will close this issue for now and can reopen if the need arises.

Reply:
The comment is still not accurate though. Maybe write something along the lines of that there are unsatisfiable dependencies on s390x to warn future developers.

Reply:
Forgot to make a comment in the recipe. It's been updated now.